managed-services/transactional-email/concepts.mdx

```diff
@@ -0,0 +1,195 @@
+---
+meta:
+  title: Transactional Email - Concepts
+  description: Learn the fundamental concepts of Scaleway Transactional Email.
+content:
+  h1: Transactional Email - Concepts
+  paragraph: Learn the fundamental concepts of Scaleway Transactional Email.
+tags: managed-services deliverability dkim dns-propagation mx-record send-api smtp transactional spf
+dates:
+  validation: 2024-11-04
+categories:
+  - managed-services
+---
+
+## Aggregate report
+
+An aggregate report (`rua`) is a periodic summary that provides data about a domain's email authentication activity. It includes information such as the volume of emails received, the percentage of emails that pass or fail DMARC checks, and the sources of failed authentication.
+
+## Blocklist
+
+Blocklist allows your to manage email addresses that generate errors, or “bounces”. You can block them improve the quality of your email deliveries, maintain your domain's reputation score, and reduce the risk of having your domains blocked by ISPs.
```

Comment:
This version of the concept only applies for custom blocklist, which are not publicly available yet.
The initial concept for blocklist is to prevent sending emails to a recipient address that was not reachable previously. The ONLY reasons we publicly have are "mailbox_not_found" and "mailbox_full", any other errors doesn't generate blocklist.